theano/compile/tests/test_modes.py

```diff
@@ -43,20 +40,14 @@ def test1(self):
         # there should be
         # - VM_Linker
         # - OpWiseCLinker (FAST_RUN)
-        # - WrapLinker ("ProfileMode")
+        # - WrapLinker
         # - PerformLinker (FAST_COMPILE)
         # - DebugMode's Linker  (DEBUG_MODE)
-        assert 5 == len(set(linker_classes_involved))
+        assert 4 == len(set(linker_classes_involved))
 
 
-class T_ProfileMode_WrapLinker(unittest.TestCase):
+class T_WrapLinker(unittest.TestCase):
     def test_1(self):
-        # First, compile a function with a new ProfileMode() object
-        # No need to call that function
-        x = T.matrix()
-        mode = ProfileMode()
-        theano.function([x], x * 2, mode=mode)
```

Comment:
I suspect that this test is not useful anymore without this part.  If that is not the case, then it is at least badly misnamed since at no point does it involve WrapLinker.